doc/lua_api.txt

```diff
@@ -2037,6 +2037,24 @@ Elements
 * End of a container, following elements are no longer relative to this
   container.
 
+### `scroll_container[<X>,<Y>;<W>,<H>;<scrollbar name>;<direction>;<scroll factor>]`
+
+* Start of a scroll_container block, all elements in a scroll_container block will
+  * take the block's coordinate as position origin,
```

Comment:
```suggestion
  * take the scroll container coordinate as position origin
```

Reply:
Why should the comma be removed?

Reply:
No particular reason. Probably happened by re-writing the line.

Reply:
I'll mark this as resolved.